.stylelintrc.js

```diff
@@ -1,3 +1,64 @@
 module.exports = {
   extends: '@wagtail/stylelint-config-wagtail',
+  rules: {
+    'scss/at-rule-no-unknown': [
+      true,
+      {
+        ignoreAtRules: [
+          'tailwind',
```

Comment:
Wonder if we should remove the tailwind references here as we (currently) don't use it.

Reply:
Yeah, we don't use it so no reason to include it.